src/cli/main/cli.py

```diff
@@ -411,11 +412,28 @@ class Config(NamedTuple):
     cache: List[Dict[str, str]]
 
 
+def _get_named_temporary_file_name() -> str:
```

Comment:
Please test this change in universe with the `m . /integrates` job

Reply:
Works!
![image](https://github.com/fluidattacks/makes/assets/50106623/a3f87520-f765-4ed4-98ba-259d8a0ff54b)